misc/msvc/game.def

```diff
@@ -1,3 +0,0 @@
-EXPORTS
```

Comment:
I have no idea. I removed the msvc folders since we dont use those projects anymore and created my own.